src/tailwind.config.ts

```diff
@@ -39,9 +33,4 @@ const config = {
       },
     },
   },
-  plugins: [prefectDesignPlugin],
-}
-
-function PrefectUiLibraryPlugins() { /* noop */ }
-
-module.exports = plugin(PrefectUiLibraryPlugins, config)
\ No newline at end of file
+} satisfies Config
```

Comment:
this is a great way to use satisfies. Gonna log that away in my brain for future use.